lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -160,12 +157,10 @@
                 return '#teams/' + this.model.get('topic_id') + '/' + this.model.get('id');
             },
             // eslint-disable-next-line no-unused-vars
-            getTopicType: function(topicId) {
-                // This function will be overrwritten in the extended class in TeamsView
+            getTopic: function(topicId) {
+                // This function will be overrwritten in the extended class in
```

Comment:
Nit: clean up the end of this sentence, it currently reads "...in the extended class in That will in turn be overwritten...".